docs/changelog.txt

```diff
@@ -1,3 +1,17 @@
+0.7.5 (2024-02-03)
+------------------------------------------------------------------------
+This release restores compatibility for stations in OpenTTD <14.
+
+Support for NewGRF additions of OpenTTD 14:
+ - Change: add constants GROUNDSPRITE_ROAD_X and GROUNDSPRITE_ROAD_Y (#307)
+ - Add: Support for build probability action CB 162 var10 0 (reverse rail vehicle). (#313)
+
```

Comment:
#315 seems to be forgotten.

Reply:
Yeah forgot to update my working copy before updating the changelog.